examples/multiple_data_source_examples/public/components/data_source_selectable_example.tsx

```diff
@@ -0,0 +1,174 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import React, { useMemo, useState } from 'react';
+import {
+  EuiBasicTable,
+  EuiBasicTableColumn,
+  EuiPageBody,
+  EuiPageContent,
+  EuiPageContentBody,
+  EuiPageHeader,
+  EuiPageHeaderSection,
+  EuiSpacer,
+  EuiText,
+  EuiTextColor,
+  EuiTitle,
+} from '@elastic/eui';
+import { MountPoint } from 'opensearch-dashboards/public';
+import { CoreStart } from 'opensearch-dashboards/public';
+import {
+  DataSourceManagementPluginSetup,
+  DataSourceSelectableConfig,
+} from 'src/plugins/data_source_management/public';
+import { ComponentProp } from './types';
+
+interface DataSourceSelectableExampleProps {
+  savedObjects: CoreStart['savedObjects'];
+  dataSourceEnabled: boolean;
+  notifications: CoreStart['notifications'];
+  setActionMenu?: (menuMount: MountPoint | undefined) => void;
+  dataSourceManagement: DataSourceManagementPluginSetup;
+}
+
+export const DataSourceSelectableExample = ({
+  savedObjects,
+  dataSourceEnabled,
+  notifications,
+  setActionMenu,
+  dataSourceManagement,
+}: DataSourceSelectableExampleProps) => {
+  const DataSourceMenu = dataSourceManagement.ui.getDataSourceMenu<DataSourceSelectableConfig>();
+  const [selectedDataSources, setSelectedDataSources] = useState<string[]>([]);
+
+  const data: ComponentProp[] = [
```

Comment:
Several files use the exact same constant data. Is it possible to simplify it so that the same content doesn't need to be copied in each file?

Reply:
The data array content is different for each component